src/go.js

```diff
@@ -1,65 +1,33 @@
-const findReachableUrls = require('find-reachable-urls');
-const readMeta = require('lets-get-meta');
 const got = require('got');
-const { tldExists } = require('tldjs');
 
 const cache = require('./utils/cache');
 const log = require('./utils/log');
 
-const getGoMeta = async (url) => {
-  const response = await got.get(url);
-  const meta = readMeta(response.body);
-
-  if (!meta['go-source']) {
-    throw new Error('go-source meta is missing');
-  }
-
-  const values = meta['go-source'].replace(/\s+/g, ' ').split(' ');
-
-  return {
-    projectRoot: values[0],
-    projectUrl: values[1],
-    dirTemplate: values[2].replace('{/dir}', ''),
-  };
-};
-
 const resolveUrl = async (url) => {
-  let goMetaConfig;
-
   const cacheKey = `go_${url}`;
   const cacheValue = await cache.get(cacheKey);
   if (cacheValue) {
     return cacheValue;
   }
 
-  if (!tldExists(`http://${url}`)) {
-    log(`"http://${url}" is not a valid hostname`);
-    return undefined;
-  }
+  const goDevUrl = `https://pkg.go.dev/${url}`;
+  const response = await got.get(goDevUrl);
 
-  try {
-    // Preferred with https
-    goMetaConfig = await getGoMeta(`https://${url}?go-get=1`);
-  } catch (err) {
-    // Fallback insecure
-    goMetaConfig = await getGoMeta(`http://${url}?go-get=1`);
+  if (response.statusCode === 404) {
+    throw new Error('Url not found');
   }
 
-  const reachableUrl = await findReachableUrls(
-    [
-      url.replace(goMetaConfig.projectRoot, goMetaConfig.dirTemplate),
-      goMetaConfig.projectUrl,
-    ],
-    { firstMatch: true },
-  );
+  const isStdLib = response.body.includes('standard library');
```

Comment:
Without this check, `import "context"` would resolve to https://cs.opensource.google/go/go, but the actual https://pkg.go.dev/context might be more interesting in this case